File: src/main/java/com/ciazhar/authserver/model/exception/AuthException.java
package com.ciazhar.authserver.model.exception;

/**
 * Created by ciazhar on 6/21/17.
 */

public class AuthException extends Exception {
    private static final long serialVersionUID = 6034102462788278056L;

    public static final String MESSAGE = "authentication error";

    public AuthException() {
        super(MESSAGE);
    }
}

File: src/main/kotlin/com/ciazhar/authserver/model/exception/AuthException.kt
package com.ciazhar.authserver.model.exception

/**
 * Created by ciazhar on 6/21/17.
 */

class AuthException : Exception(MESSAGE) {
    companion object {
        private val serialVersionUID = 6034102462788278056L

        val MESSAGE = "authentication error"
    }
}
